src/LanguageLine.php

```diff
@@ -52,12 +54,12 @@ public static function getCacheKey(string $group, string $locale): string
      *
      * @return string
      */
-    public function getTranslation(string $locale): string
+    public function getTranslation(string $locale)
```

Comment:
Let's make this a nullable typehint. PHP 7.1 may be required in composer.json.

Reply:
Yes, it indeed needs PHP 7.1. Changed.